src/pages/Information/Information.jsx

```diff
@@ -1,32 +1,54 @@
 import React from "react";
+import { FormattedMessage } from "react-intl";
 
 const Information = () => (
   <>
     <div data-test="information-display">
-      <h2 data-test="heading-display">What is HacktoberFest?</h2>
+      <h2 data-test="heading-display">
+        <FormattedMessage
+          id="Info.first-header"
+          defaultMessage="What is HacktoberFest?"
+        />
+      </h2>
       <p data-test="para-display">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
-        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
-        mollit anim id est laborum.
+        <FormattedMessage
+          id="Info.first-paragraph"
+          defaultMessage="Lorem ipsum..."
+        />
       </p>
-      <img data-test="image-display" src="" alt="hacktoberfest" />
+      <img
+        data-test="image-display"
+        src="https://pngimg.com/uploads/pumpkin/pumpkin_PNG86719.png"
+        alt="hacktoberfest"
+        style={{ height: "100px" }}
+      />
     </div>
     <div>
-      <h2 data-test="heading-display">When & Where</h2>
+      <h2 data-test="heading-display">
+        <FormattedMessage
+          id="Info.second-header"
+          defaultMessage="When & Where"
+        />
+      </h2>
       <p data-test="para-display">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua.
+        <FormattedMessage
+          id="Info.second-paragraph"
+          defaultMessage="Lorem ipsum..."
+        />
       </p>
     </div>
     <div>
-      <h2 data-test="heading-display">Come join the FUN!</h2>
+      <h2 data-test="heading-display">
+        <FormattedMessage
+          id="Info.third-header"
+          defaultMessage="Come join the FUN!"
+        />
+      </h2>
       <p data-test="para-display">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua.{" "}
+        <FormattedMessage
+          id="Info.third-paragraph"
+          defaultMessage="Lorem ipsum..."
+        />
```

Comment:
🔧 Can you extract all of the data to an array of objects, then loop through it to display the values?

Reply:
Created an issue for this so we can implement later.
https://github.com/aaron-org/snapit-hacktoberfest/issues/40